### PR TITLE
chore: convert all examples to yaml

### PR DESCRIPTION
**Pull request by XVincentX**
There is currently no reason to have the petstore json example and the oas3 one in JSON. Remove and unify.

### CODE REVIEW

File: packages/http-server/src/__tests__/server.oas.spec.ts [hunk 1]
```diff
@@ -206,7 +206,7 @@ describe.each([['petstore.no-auth.oas2.yaml', 'petstore.no-auth.oas3.yaml']])('s
     });
 
     // oas2 does not support overriding servers and named examples
-    if (file === 'petstore.oas3.json') {
+    if (file === 'petstore.no-auth.oas3.yaml') {
```
Comment (marcelltoth) [inline packages/http-server/src/__tests__/server.oas.spec.ts]: I think you'd need to update lines 178-179 to _.no-auth_ too, don't you?

Comment (XVincentX) [reply packages/http-server/src/__tests__/server.oas.spec.ts]: I've put it in a constant so we stop with these magic strings.

**Comment by XVincentX** [comment]
> Where do these *.no-auth.oas3.yaml files come from?

We have them in the repo:

https://github.com/stoplightio/prism/blob/master/packages/http-server/src/__tests__/fixtures/petstore.no-auth.oas3.yaml

https://github.com/stoplightio/prism/blob/master/packages/http-server/src/__tests__/fixtures/petstore.no-auth.oas2.yaml

**Comment by marcelltoth** [comment]
> 
> 
> > Where do these *.no-auth.oas3.yaml files come from?
> 
> We have them in the repo:

I don't know what I was looking at 😅

**Pull request merged**